hieunc-guestbook-dojo/static/js/guestbook/widget/_WidgetGuestBookDel.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Created by hieunc on 25/11/2015.
+ */
+
+define([
+    "dojo/_base/declare",
+    "dijit/_WidgetBase",
+    "dijit/_TemplatedMixin",
+    "dojo/text!./templates/guestbook_del.html",
+    "dojo/request",
+    "dojo/cookie"
+], function (declare, _WidgetBase, _TemplatedMixin, template, request, cookie) {
+    return declare("GuestBookDel", [_WidgetBase, _TemplatedMixin], {
```

Comment:
- đặt tên class theo cấu trúc như sau: `path/to/file/ClassName`
- tên class trùng với tên file (trừ trường hợp là SubClass)

ví dụ: `guestbook/widget/_WidgetGuestBookDel`